src/homepage.rs

```diff
@@ -83,7 +86,7 @@ mod tests {
         let template = r#"
 <table>
 {% for member in members -%}
-<tr><td>{{ member.name }}</td><a href="{{ member.website.href }}">{{ member.website.authority }}</a></td></tr>
+<tr><td>{{ member.name }}</td><a href="{{ base_addr.scheme }}://{{ base_addr.authority }}{{ base_addr.path_and_query }}visit?member={{ member.website.authority }}">{{ member.website.authority }}</a></td></tr>
```

Comment:
Can we just keep the `href` field? It makes things cleaner, and it feels safer to do URL manipulation in the Rust code than in the templates.

Reply:
You're right; I just realized that I didn't understand how your code was meant to work when I got rid of it.